### ISSUE TITLE
Maintained?

### ISSUE THREAD

**Issue by geertbrit** [procurios/ElementQueries#1]
coming from http://marcj.github.io/css-element-queries/ looking for a vanilla javascript variant with ability to polyfill querySelectorAll. This seems like the thing. 

Wondering: is this lib still used in prod? Not seeing any progress, although it may just be done of course.

**Comment by pesla** [comment]
@gebrits Yes, this is an active component of our platform and used by ~10.000 - ~100.000 users every day. I haven't had the opportunity to update this repository with a roadmap, but the current codebase is stable in the context of how we use it (eg. in our platform).

Feel free to try if it fits in your stack and report back with any issues. Happy to help!
